### DISCUSSION TITLE
Add items to workbench by default

### DISCUSSION THREAD

**Discussion by AndreasAtakan** [TerriaJS/terriajs#6314] [General] [answered]
Hi!

I want to add some catalogue items to the workbench when the user loads the page.

I found a setting for init.json called "workbench" (which takes an array of strings, presumably catalogue item-names), but I can't get it to work..

Any help on this?
Thanks in advance

**Comment by AndreasAtakan** [comment]
@nf-s ?

**Comment by zoran995** [comment, marked as answer]
Workbench is array of item ids, every catalog item can have it, if not specified terria will autogenerate id but I would specify them manually and then use to populate workbench
